API-1251: Update documentation to get categories filtered by parent (#403)
akeneo/pim-api-docs: content/rest-api/filter.md

```
/api/rest/v1/attributes?search={"type":[{"operator":"IN","value":["pim_catalog_simpleselect","pim_catalog_multiselect"]}]}
```

## Filter categories

When you request specific categories, you can use filters to select only the ones you want. 

> ### By category parent

Reviewer: ```suggestion
### By parent category
```

> ### By categories parents

Reviewer 2: ```suggestion
### On parent categories
```

::: availability versions=Serenity editions=CE,EE

> You can filter the categories by their parents.

Reviewer 2: ```suggestion
You can filter the categories on their parents. When you add a parent category in the filter, the response will contain all its child sub-categories. 
```

Below you will find the operator allowed to filter on this property, as well as the corresponding type of value required in the `search` query parameter.

| Operator | Allowed value type | Filter description |
| ----------------- | -------------- | ------------------ |

> | `IN` | an array of [existing categories parents](/concepts/catalog-structure.html#category) | Only returns the categories of the given parents |

Reviewer 2: ```suggestion
| `=` | a string containing an [existing parent category](/concepts/catalog-structure.html#category) | Only returns the sub-categories of the given parent |
```

#### Example

> To get the categories of parents `categoryA` and `categoryB`, you can use the following URL.

Reviewer 2: ```suggestion
To get the child categories of parents `categoryA` and `categoryB`, you can use the following URL.
```

```

> /api/rest/v1/categories?search={"type":[{"operator":"IN","value":["categoryA","categoryB"]}]}

Reviewer 2: ```suggestion
/api/rest/v1/categories?search={"parent":[{"operator":"IN","value":["categoryA","categoryB"]}]}
```

> To get the child categories of parents `categoryA` and `categoryB`, you can use the following URL.

Reviewer 2: ```suggestion
To get the child categories of the parent category `categoryA`, you can use the following URL.
```

> /api/rest/v1/categories?search={"parent":[{"operator":"IN","value":["categoryA","categoryB"]}]}

Reviewer 2: ```suggestion
/api/rest/v1/categories?search={"parent":[{"operator":"=","value":"categoryA"}]}
```
